packages/keyring-eth-hd/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - **BREAKING:** The method signature for `signTypedData` has been changed ([#224](https://github.com/MetaMask/accounts/pull/224))
   - The method now accepts a `TypedDataV1` object when `SignTypedDataVersion.V1` is passed in the options, and `TypedMessage<Types>` when other versions are requested.
+- **BREAKING:** The `HdKeyring` class now extends `Keyring` from `@metamask/keyring-utils`
+  - The `deserialize` method does not accept `Buffer` mnemonic anymore
```

Comment:
Is it worth noting these as well?

``` suggestion
  - The `deserialize` method does not accept `Buffer` mnemonic anymore
  - The options argument to `deserialize` is no longer optional
  - `getAccounts` is now `async`

Reply:
Also is it worth mentioning that `deserialize` now takes a `SerializedBuffer`? Or was that already the case?

Reply:
> Also is it worth mentioning that deserialize now takes a SerializedBuffer? Or was that already the case?

It was already the case, but now the types are more accurate